ui/src/app/views/cohort-list/component.ts

```diff
@@ -0,0 +1,73 @@
+import {Component, OnDestroy, OnInit} from '@angular/core';
+import {ActivatedRoute, Router} from '@angular/router';
+
+import {WorkspaceData} from 'app/resolvers/workspace';
+
+import {
+  Cohort,
+  CohortsService,
+  Workspace,
+  WorkspaceAccessLevel,
+} from 'generated';
+
+
+@Component({
+  styleUrls: ['../../styles/buttons.css',
+    '../../styles/cards.css',
+    './component.css'],
+  templateUrl: './component.html',
+})
+export class CohortListComponent implements OnInit, OnDestroy {
+  accessLevel: WorkspaceAccessLevel;
+  awaitingReview: boolean;
+  cohortList: Cohort[] = [];
+  workspace: Workspace;
+  cohortsLoading = true;
+  cohortsError = false;
+  wsNamespace: string;
+  wsId: string;
+
+  constructor(
+    private route: ActivatedRoute,
+    private cohortsService: CohortsService,
+    private router: Router,
+  ) {
+    const wsData: WorkspaceData = this.route.snapshot.data.workspace;
+    this.workspace = wsData;
+    this.accessLevel = wsData.accessLevel;
+    const {approved, reviewRequested} = this.workspace.researchPurpose;
+    this.awaitingReview = reviewRequested && !approved;
+  }
+
+  ngOnInit(): void {
+    this.wsNamespace = this.route.snapshot.params['ns'];
+    this.wsId = this.route.snapshot.params['wsid'];
+    this.cohortsService.getCohortsInWorkspace(this.wsNamespace, this.wsId)
+      .subscribe(
+        cohortsReceived => {
+          for (const coho of cohortsReceived.items) {
```

Comment:
Can we use arrow function or something like  cohortsReceived.map(function(cohorts){return cohorts.item;})?

Reply:
This does work!